controllers/scopetemplate_controller.go

```diff
@@ -71,21 +80,39 @@ func (r *ScopeTemplateReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	log.Log.Info("Reconciling ScopeTemplate")
 
 	// get the scope template
-	st := &operatorsv1.ScopeTemplate{}
-	if err := r.Client.Get(ctx, req.NamespacedName, st); err != nil {
-		return ctrl.Result{}, err
+	existingSt := &operatorsv1.ScopeTemplate{}
```

Comment:
We called this object existingIN in the scopeInstance controller, let's update that to existingST.

Reply:
It should be `existingIn` I think in other areas of the code we have `existingCRB` or `existingRB`. Would you prefer that we do something like `existingSI` (SI for `ScopeInstance`) or change the instances of `existingCRB` to `existingCrb`?